CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Unreleased
 
 ### Changes
+* [IMPROVEMENT] Add mobile vitals frequency configuration. See [#851][]
 
 # 1.11.0-rc1 / 18-05-2022
 
 ### Changes
 
 * [IMPROVEMENT] Allow manually tracked resources in RUM Sessions to detect first party hosts. See [#837][]
-* [IMPROVEMENT] Add tracing sampling rate. See [#851][]
+* [IMPROVEMENT] Add tracing sampling rate. See [#876][]
```

Comment:
This seems wrong, the current PR is #876 and tracing sampling was #851.

Reply:
oops!